core/src/integration-test/java/com/scalar/db/storage/cassandra/ConsensusCommitAdminIntegrationTestWithCassandra.java

```diff
@@ -17,4 +19,20 @@ protected String getSystemNamespaceName(Properties properties) {
         .getSystemNamespaceName()
         .orElse(DatabaseConfig.DEFAULT_SYSTEM_NAMESPACE_NAME);
   }
+
+  @Override
+  protected String getCoordinatorNamespaceName(String testName) {
+    return new ConsensusCommitConfig(new DatabaseConfig(getProperties(testName)))
+        .getCoordinatorNamespace()
+        .orElse(Coordinator.NAMESPACE);
+  }
+
+  @Override
+  protected boolean isGroupCommitEnabled(String testName) {
+    return new ConsensusCommitConfig(new DatabaseConfig(getProperties(testName)))
+        .isCoordinatorGroupCommitEnabled();
+  }
+
+  @Override
+  protected void extraCheckOnCoordinatorTable() {}
```

Comment:
The default check doesn't work. Maybe it's related to https://github.com/scalar-labs/scalardb/blob/f57fe48151915819ea659646cb9406d2e6c2b18d/core/src/integration-test/java/com/scalar/db/storage/cassandra/CassandraAdminRepairIntegrationTest.java#L26-L29...?